Uno.Gallery/Uno.Gallery.Shared/Views/SamplePages/InfoBarSamplePage.xaml

```diff
@@ -1,20 +1,39 @@
-﻿<Page x:Class="Uno.Gallery.Views.SamplePages.InfoBarSamplePage"
-	  xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
-	  xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
-	  xmlns:local="using:Uno.Gallery.Views.SamplePages"
-	  xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
-	  xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
-	  xmlns:sample="using:Uno.Gallery"
-	  xmlns:smtx="using:ShowMeTheXAML"
-	  xmlns:winui="using:Microsoft.UI.Xaml.Controls"
-	  mc:Ignorable="d">
+<Page x:Class="Uno.Gallery.Views.SamplePages.InfoBarSamplePage"
+      xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+      xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+      xmlns:local="using:Uno.Gallery.Views.SamplePages"
+      xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+      xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+      xmlns:sample="using:Uno.Gallery"
+      xmlns:smtx="using:ShowMeTheXAML"
+      xmlns:winui="using:Microsoft.UI.Xaml.Controls"
+      mc:Ignorable="d">
 
-	<Grid Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
-		<sample:SamplePageLayout>
-			<sample:SamplePageLayout.MaterialTemplate>
-				<DataTemplate>
-					<StackPanel Spacing="20"
-								Margin="0,20,0,0">
+    <Page.Resources>
+        <!-- Button Styles -->
```

Comment:
Is anything lost by not basing these new styles on `TextButtonStyle`?
Such as Min sizes and different focus visuals?
Is this intentional/deliberate? All you've done is change the styles to use different colors and with no explanation of why.